File: MultiLanguageSandbox/src/test/deps/haskell/41.hs
absoluteValue :: Int -> Int
absoluteValue x
    | abs x > 10000 = error "Input absolute value exceeds 10000"
    | otherwise     = abs x
check :: (Int -> Int) -> IO ()
check absoluteValueFunc = do
  let testCases = 
        [ (-3, 3),
          (5, 5),
          (0, 0),
          (-10000, 10000),
          (9999, 9999)
        ]

  mapM_ (\(number, expected) -> testCase number expected) testCases
  where
    testCase number expected = do
      let result = absoluteValueFunc number
      putStrLn $
        "Input: " ++ show number ++
        ", Expected: " ++ show expected ++ ", Result: " ++ show result ++
        if result == expected then " (Pass)" else error "Test failed. Exiting..."

main :: IO ()
main = check absoluteValue

File: MultiLanguageSandbox/src/test/deps/F#/41.fs

let calculateExpression (a: int) (b: int) (c: int): int =
    (a + b) * c



let check () =
    if calculateExpression 1 2 3 <> 9 then
        failwith "Test Case 1 failed"
    if calculateExpression -1 2 -3 <> -3 then
        failwith "Test Case 2 failed"
    if calculateExpression 0 0 1 <> 0 then
        failwith "Test Case 3 failed"
    if calculateExpression 100 -50 2 <> 100 then
        failwith "Test Case 4 failed"
    if calculateExpression -100 50 -2 <> 100 then
        failwith "Test Case 5 failed"

check ()